lib/core/src/sdk.rs

```diff
@@ -142,6 +142,18 @@ impl LiquidSdkBuilder {
         self
     }
 
+    pub fn default_persister(&self, sync_enabled: bool) -> Result<Arc<Persister>> {
```

Comment:
Perhaps it makes sense to move this function to the Persister struct?
We can have two methods:
```
new_fs_persister(workingDir, syncEnabled)
new_memory_persisiter(syncEnabled)
```

It will also allow us to move the calls of `get_working_dir` and `std::fs::create_dir_all(&main_db_dir)` to the corresponding function and exclude/include the whole functions based on wasm build or not.

Reply:
I wasn't keen on having a specific in-memory constructor for the persister because of us not having control over when it's in-memory or not, but excluding the constructors on the appropriate environment solves that issue, thanks!

I applied this on [ac846b4](https://github.com/breez/breez-sdk-liquid/pull/792/commits/ac846b4d4f84baa214cb4a66bb085bfd5b2f8dc2). Still kept using the path for the in-memory version because `sqlite-wasm-rs` allows using the same in-memory db if we provide the same path. Not sure if we have an immediate need for this, but it's there.